src/EntityFramework6.Npgsql/NpgsqlMigrationSqlGenerator.cs

```diff
@@ -500,18 +500,50 @@ protected virtual void Convert(DropPrimaryKeyOperation dropPrimaryKeyOperation)
         #region Misc functions
 
         /// <summary>
+        /// Quotes an identifier for Postgres SQL and appends it to a string builder
+        /// </summary>
+        /// <param name="builder"> The string builder used for building the query. </param>
```

Comment:
nit, style: don't leave unnecessary spaces around the sentence (consistency). Use cref.